src/dmd/mars.d

```diff
@@ -406,6 +406,12 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         return result;
     }
 
+    if (params.mixinFile)
+    {
+        params.mixinOut = cast(OutBuffer*)calloc(1, OutBuffer.sizeof);
```

Comment:
Really should not call calloc() and not check the return value.

Reply:
This line has just moved in this commit (line 1716 originally).